Sources/App/Views/Home/HomeIndex+View.swift

```diff
@@ -41,6 +41,7 @@ enum HomeIndex {
                     .div(
                         .class("inner"),
                         .h3("The place to find Swift packages."),
+                        .structuredData(IndexSchema()),
```

Comment:
Let's keep this in the `bodyComments` for now. That method still needs a rename, but it's a better place for this data.

In an ideal world, I think this would fit best just *before* the `</body>` tag. Maybe a method named `bodyPreClosing` (ugh that's an awful name), `bodyFooter` (ugh, makes it seem like it's visual), or something similar (suggestions welcome)

Maybe just `andFinally...` 😂

Reply:
Didn't see it when I added it, only just realised it's an optional override from parent class. Moved 🙂

Reply:
I moved the other one for the package data there too, but I just pushed it up in a commit so it wasn't super obvious. Thanks for moving it!

How about just `preBody` and `postBody` to replace `bodyComments` and add the one at the end?

Reply:
Probably a separate PR though, no need to corrupt this one with that change. I'm just still thinking about it 😂

Reply:
Aye happy to add these new functions as part of a separate PR and apply it to both schemas. Will create `preBody` and `postBody` to go at the top and bottom respectively.

Reply:
On second thoughts, got bored - have done it as part of this PR 😛  (sorry)